CodeEditModules/Modules/CodeFile/src/CodeEditor.swift

```diff
@@ -63,8 +63,8 @@ struct CodeEditor: NSViewRepresentable {
         scrollView.documentView = textView
         scrollView.verticalRulerView = LineGutter(
             scrollView: scrollView,
-            width: 45,
-            font: .monospacedDigitSystemFont(ofSize: 11, weight: .regular),
+            width: 37,
+            font: NSFont(name: "SF Mono", size: 11) ?? .monospacedSystemFont(ofSize: 11, weight: .regular),
```

Comment:
Isn't just `monospacedSystemFont(ofSize:weight: )` plenty since it is SF Mono by default?

Reply:
Actually not... Because if you compare it to xcode you see the difference.